src/net/KNet/KNetCore.cs

```diff
@@ -75,6 +75,24 @@ public override IEnumerable<IArgumentMetadata> CommandLineArguments
                         Default = Const.DefaultLogPath,
                         Help = "The path where log will be stored.",
                     },
+                    new ArgumentMetadata<string>()
+                    {
+                        Name = CLIParam.DisableJMX,
+                        Type = ArgumentType.Single,
+                        Help = "Disable JMX. Default is true.",
+                    },
+                    new ArgumentMetadata<string>()
+                    {
+                        Name = CLIParam.EnableJMXAuth,
+                        Type = ArgumentType.Single,
+                        Help = "Enable authenticate on JMX. Default is false",
```

Comment:
Not clear, it is a switch without values

Reply:
Hided for now